.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: eFwTmwDziLPmodbfNNHxHIJOHE5aG4Rmp
```

Comment:
**Note: Do not make your repo token public.**
https://coveralls.io/github/Purple-Ninja/monga

No token key seems required for basic integration of travis-ci and github.

But if you found token key still needed,
follow the key encryption of travis ci, https://docs.travis-ci.com/user/encryption-keys/

Anyway, you need to regenerate the token since its already public.